rsconnect/main.py

```diff
@@ -460,7 +460,10 @@ def deploy_api(name, server, api_key, insecure, cacert, entrypoint, exclude, new
         check_server_capabilities(connect_server, checks)
 
     with cli_feedback('Inspecting Python environment'):
-        _, environment = get_python_env_info(directory, python, not conda, force_generate)
+        root = directory
+        if not root.endswith('/'):
```

Comment:
Same as above: using trailing slashes is a little less consistent/definitive than stat'ing whether it's a directory (`isdir`) IMO.  Subjective, but just wanted to offer an alternative... your call.

Reply:
Fixed differently.

Reply:
All good, you do you.